### PR TITLE
Print message if there are no input files for fix subcommand (#385)

### LINKED ISSUE #385: Print a message "Nothing to fix"
### Description

Print a message "Nothing to fix" in the absence of `.env` files for fix.

### Implementation

**Step 1:**

Add a method for [FixOutput](https://github.com/dotenv-linter/dotenv-linter/blob/master/src/common/output/fix.rs#L14) to print the message "Nothing to fix" only if `is_quiet_mode == false`.

**Step 2:**

Call the method from the step 1 when [lines_map](https://github.com/dotenv-linter/dotenv-linter/blob/master/src/lib.rs#L54) is empty.

**Step 3:**

Add integration tests [here](https://github.com/dotenv-linter/dotenv-linter/blob/master/tests/output/fix.rs).

### Example

There aren't `.env` files in the current directory:

```sh
$ dotenv-linter fix
Nothing to fix
```

### PR DESCRIPTION
**Pull request by mdycz**
Closes #385

#### ✔ Checklist:

- [x] Commit messages have been written in [Conventional Commits](https://www.conventionalcommits.org) format;
- [x] This PR has been added to [CHANGELOG.md](https://github.com/dotenv-linter/dotenv-linter/blob/master/CHANGELOG.md) (at the top of the list);
- [x] Tests for the changes have been added (for bug fixes / features);

### CODE REVIEW

**Review by mstruebing — DISMISSED** [review]
Thank you very much, for me it looks good!

File: src/common/output/fix.rs [hunk 1]
```diff
@@ -54,4 +54,13 @@ impl FixOutput {
             println!();
         }
     }
+
+    /// Prints no files found message
+    pub fn print_no_files_message(&self) {
```
Comment (mgrachev) [inline src/common/output/fix.rs:59]:
```suggestion
    pub fn print_nothing_to_fix(&self) {
```

File: src/common/output/fix.rs [hunk 2]
```diff
@@ -54,4 +54,13 @@ impl FixOutput {
...
+        if self.is_quiet_mode || self.files_count > 0 {
+            return;
+        }
+
+        println!("\nNothing to fix");
```
Comment (mgrachev) [inline src/common/output/fix.rs:64]:
```suggestion
        println!("Nothing to fix");
```

File: src/lib.rs [hunk 3]
```diff
@@ -49,9 +49,11 @@ pub fn check(args: &clap::ArgMatches, current_dir: &PathBuf) -> Result<usize, Bo
 pub fn fix(args: &clap::ArgMatches, current_dir: &PathBuf) -> Result<(), Box<dyn Error>> {
     let mut warnings_count = 0;
     let lines_map = get_lines(args, current_dir);
+    let output = FixOutput::new(args.is_present("quiet"), lines_map.len());
 
     // Nothing to fix
     if lines_map.is_empty() {
+        output.print_no_files_message();
```
Comment (mgrachev) [inline src/lib.rs:56]:
```suggestion
        output.print_nothing_to_fix();
```

**Review by mgrachev — COMMENTED** [review]
@mdycz 👋 Thank you for your contribution 🚀

I have left some comments. Please take a look at them 👀

File: src/common/output/fix.rs (hunk 1 above)
Comment (mdycz) [reply src/common/output/fix.rs]: I have renamed the function and removed that unnecessary `\n`, thanks for CR :)

**Review by DDtKey — APPROVED** [review]

**Review by mgrachev — APPROVED** [review]

**Comment by mgrachev** [comment]
@mdycz Thank you for your help ✌️

🙏 If it’s not difficult for you, please support the project - star on GitHub ⭐️

**Pull request merged**